Source/Core/Core/AchievementManager.cpp

```diff
@@ -497,6 +517,55 @@ void AchievementManager::FilereaderClose(void* file_handle)
   delete static_cast<FilereaderState*>(file_handle);
 }
 
+void AchievementManager::LoadDefaultBadges()
+{
+  std::lock_guard lg{m_lock};
+
+  std::string directory = File::GetSysDirectory() + DIR_SEP + RESOURCES_DIR + DIR_SEP;
+
+  if (m_default_player_badge.data.empty())
+  {
+    if (!LoadPNGTexture(&m_default_player_badge,
+                        directory + static_cast<std::string>(DEFAULT_PLAYER_BADGE_FILENAME)))
```

Comment:
Err what? Can you even cast a string_view to a string...? This feels super sketchy, please just use a constructor or `fmt::format` or *anything else* here.